controller/src/main/java/org/jboss/as/controller/operations/global/ReadFeatureDescriptionHandler.java

```diff
@@ -388,6 +389,7 @@ private ModelNode describeFeature(final Locale locale, final ImmutableManagement
             }
         }
         Set<String> capabilities = new HashSet<>();
+        Set<String> additionalPackages = new TreeSet<>();
```

Comment:
The order of packages is also taken into account when the packages are installed, i.e. it will be the order in which they are installed (unless there are dependencies that require a different ordering). Although for modules (every module is represented as a package in a feature-pack) it's practically irrelevant.

Reply:
@jmesnil It was just to be consistent. I'll add a commit to order the others.